test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,222 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/colorado_lottery'
+require './lib/game'
+require './lib/contestant'
+
+class ColoradoLotteryTest < Minitest::Test
+
+  def setup
+    @lottery = ColoradoLottery.new
```

Comment:
Great job using a setup method to help speed you up. You could also have added game_interests to contestants in here to DRY this up.